resources/assets/coffee/react/mp-history/score.coffee

```diff
@@ -47,7 +47,9 @@ class MPHistory.Score extends React.Component
         div className: 'mp-history-player-score__info-box mp-history-player-score__info-box--stats',
           div className: 'mp-history-player-score__stat-row mp-history-player-score__stat-row--first',
             div className: 'mp-history-player-score__mods-box',
-              el Mods, mods: @props.score.mods, reversed: true
+              el Mods,
```

Comment:
something should be done with this in mobile. At least the username need to be stay visible.